java-security-ocsp-property

```diff
@@ -0,0 +1 @@
+ocsp.enable=true
```

Comment:
Do we need this file - if its always true and only used in `run-ocsp-test.sh`

Reply:
See https://dzone.com/articles/how-override-java-security for a description of this property